frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -175,20 +175,20 @@ class PopupCreateFile extends React.Component {
 
     this.setState({ uploadStarted: true })
 
-    Promise.all(state.fileToUploadList.map(async (file) => {
-      const xhr = await this.postFile(file)
-      return this.handleFileUploadEnd(xhr, file)
-    })).catch(() => {
-      this.sendGlobalFlashMessage(props.t('Error while creating file'))
-    }).then(uploadedFiles => {
-      this.handleAllFileUploadEnd(uploadedFiles)
-    })
+    Promise.all(state.fileToUploadList.map((file) => this.postFile(file)))
```

Comment:
Promise.all(state.fileToUploadList.map((file) => ({
  xhr: this.postFile(file),
  file: file
})))